client/components/editor/Sidebar/Navigation.vue

```diff
@@ -64,12 +66,22 @@ export default {
     };
   },
   computed: {
-    activityTree: vm => toTreeFormat(vm.activities, vm.editableActivityConfigs),
-    activityConfigs: vm => getOutlineLevels(vm.repository.schema),
+    activityConfigs: vm => vm.$schemaService.getOutlineLevels(vm.repository.schema),
     activityTypes: vm => vm.activityConfigs.map(it => it.type),
     editableTypes: vm => vm.editableActivityConfigs.map(it => it.type),
     editableActivityConfigs() {
-      return this.activityConfigs.filter(it => isEditable(it.type));
+      return this.activityConfigs.filter(it => this.$schemaService.isEditable(it.type));
```

Comment:
Some of these seem no longer needed 🤔 ? (editableActivityConfigs, editableTypes, activityTypes, activityConfigs)

Reply:
Yea, `activityTypes` is not used anymore

Reply:
Since schema service has isEditable method, isActivityEditable is also not needed in this form, which also makes editableTypes and editableActivityConfigs obsolete ? 🤔

Reply:
You are right, I only checked usage instead of the actual need for a particular code 🤦‍♂️.